src/main/java/org/repodriller/CommitVisitorPersistenceMechanisMap.java

```diff
@@ -0,0 +1,83 @@
+package org.repodriller;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.log4j.Logger;
+import org.repodriller.domain.Commit;
+import org.repodriller.persistence.PersistenceMechanism;
+import org.repodriller.scm.CommitVisitor;
+import org.repodriller.scm.SCMRepository;
+
+public class CommitVisitorPersistenceMechanisMap {

```

Comment:
there's a typo here.

I think this name is a bit complicated. What about simply `CommitVisitors` ?